src/Tile/RadioTile.svelte

```diff
@@ -23,10 +26,16 @@
   /** Specify a name attribute for the input */
   export let name = "";
```

Comment:
Maybe also switch the default to `undefined` so the attribute is not added without value.